controllers/index.js

```diff
@@ -0,0 +1,21 @@
+const Contact = require('./contacts.js')
+
+const getAllContacts = async () => Contact.find()
+
+const getContactById = async contactId => Contact.findById(contactId)
```

Comment:
```suggestion
const getContactById = async contactId => Contact.findById(contactId)
```

потрібно додати перевірку на те чи знайдено контакт